modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/util/AccountBriefUtil.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.admin.user.internal.dto.v1_0.util;
+
+import com.liferay.account.constants.AccountConstants;
+import com.liferay.account.model.AccountEntry;
+import com.liferay.account.service.AccountRoleLocalService;
+import com.liferay.headless.admin.user.dto.v1_0.AccountBrief;
+import com.liferay.headless.admin.user.dto.v1_0.RoleBrief;
+import com.liferay.petra.function.transform.TransformUtil;
+import com.liferay.portal.kernel.model.Role;
+import com.liferay.portal.vulcan.dto.converter.DTOConverterContext;
+import com.liferay.portal.vulcan.util.LocalizedMapUtil;
+
+/**
+ * @author Crescenzo Rega
+ */
+public class AccountBriefUtil {
+
+	public static AccountBrief toAccountBrief(
+			AccountEntry accountEntry,
+			AccountRoleLocalService accountRoleLocalService,
+			DTOConverterContext dtoConverterContext, long userId)
+		throws Exception {
+
+		if ((accountEntry == null) ||
+			(accountEntry.getAccountEntryId() ==
+				AccountConstants.ACCOUNT_ENTRY_ID_DEFAULT)) {
+
+			return null;
+		}
+
+		return new AccountBrief() {
+			{
+				setExternalReferenceCode(
+					accountEntry::getExternalReferenceCode);
+				setId(accountEntry::getAccountEntryId);
+				setName(accountEntry::getName);
+				setRoleBriefs(
+					() -> TransformUtil.transformToArray(
+						accountRoleLocalService.getAccountRoles(
+							accountEntry.getAccountEntryId(), userId),
+						accountRole -> new RoleBrief() {
+							{
+								setExternalReferenceCode(
+									accountRole::getExternalReferenceCode);
+								setId(accountRole::getAccountRoleId);
+								setName(accountRole::getRoleName);
+								setName_i18n(
+									() -> {
+										Role role = accountRole.getRole();
+
+										return LocalizedMapUtil.getI18nMap(
+											dtoConverterContext.
+												isAcceptAllLanguages(),
+											role.getTitleMap());
+									});
+							}
+						},
+						RoleBrief.class));
```

Comment:
mmm... I don't want to have this always: example
when I export an organization I don't care about the roles of the accounts, in other scenario instead I need them. Where is this util used?